subfunctions/demo_CaBBI_FHN

```diff
@@ -0,0 +1,236 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% This script inverts the FHN model for the given fluorescence trace using
+% CaBBI method: [ Rahmati. et al. 2016, PLoS Comput Biol 12(2) ]
+%
+% To user:
+% I) in Step 1, determine the directory to the fluorescence trace (a row vector)
+% II) in Step 2, determine the sampling rate (Hz) used for recording the fluorescence trace
+% III) run the code
+% IV) the main outputs of the CaBBI are listed in Step 11. 
+%
+% Further options:
+% 1) In Step 10, if necessary, you can change the spike-detection threshold which is used to extract 
+%    spike-times from inferred voltage traces. Such change may increase the spike-detection accuracy. 
+% 2) In Step 6, if necessary, you can change some internal parameters of the optimization process.
+% 3) For further description of the toolbox options, please see:
+%    http://mbb-team.github.io/VBA-toolbox/wiki/Controlling-the-inversion-using-VBA-options/
+%    http://mbb-team.github.io/VBA-toolbox/wiki/VBA-output-structure/
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+clc; clear all; close all
```

Comment:
What I had in mind is simply to start the code with
`function demo_CaBBI_FHN`
such that you do not need the `clear all` at the beginning that might delete important results from the workspace of the user